### ISSUE TITLE
Paper-checkbox label does not wrap correctly

### ISSUE THREAD

**Issue by WoodyWoodsta** [PolymerElements/paper-checkbox#103]
Sizing the `paper-checkbox` element in any way such that the label starts to need to wrap causes the label to flow outside of the width of the element. The label appears to not wrap for a width that of the checkbox div outside of the entire element. In other words, getting rid of the checkbox div fixes the issue.

Below is the usage of the checkbox in my app (excuse the label :stuck_out_tongue_closed_eyes:):
![screenshot from 2016-01-31 22 28 49](https://cloud.githubusercontent.com/assets/8268040/12704838/7d62d8a8-c86c-11e5-9d59-a7a63aec937f.png)

And here is a [reduced test case](http://jsbin.com/patiko/2/edit?html,css,output).

The issue is present in latest Firefox security release (44.0), as well as Chrome 50.0.2633.3. I don't have access to others.

**Comment by notwaldorf** [comment]
This is true. I wonder if this could be fixed with a flex box, and forcing the items to align at the top

**Comment by danbeam** [comment]
@notwaldorf flex box would be well, @hcarmona's fix would probably work as well, though

fyi: this is affecting chrome://md-settings

**Comment by danbeam** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=620002#c11

**Comment by notwaldorf** [comment]
cc @bicknellr -- I am not `paper-checkbox`'s keeper anymore :)

**Comment by danbeam** [comment]
@tjsavage

**Comment by davidmaxwaterman** [comment]
this is still a problem - paper-checkbox#1.4.2 - the label only wraps when it touches the label of the checkbox next to it:

![paper-checkbox-wrapping](https://user-images.githubusercontent.com/975698/40537922-136e41ae-6008-11e8-8f52-4e77b4b4cd71.png)
